x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -124,6 +133,7 @@ export class Simulator {
       yield mapper();
       await new Promise((resolve) => {
         setTimeout(() => {
+          this.forceAutoSizerOpen();
```

Comment:
:grey_question: I was thinking you'd just call this when you need it in another public method (like `clickRelatedButton`) rather than inlining it here.

Reply:
why?

Reply:
Few thoughts on this:
* AutoSizer provides behavior we don't even want. We'll probably stop using EuiSelectable in 7.10.
* We may end up accidentally using AutoSizer in the future since we use Eui. Eui could add it in places or we could start using new Eui stuff.
* This is only needed because of limitations with JSDOM anyway.

With the stuff above in mind:
* I don't think devs should ever have to think about this in order to write tests
* Tests shouldn't be coupled to this

All that being said, putting this in `map` seems to satisfy all the constraints. Any reason why it shouldn't be there?

Reply:
Map is at the center of this 'new approach' to testing. Adding something that only has to be there for some edge-case-set of tests seems like it would
1) Make the core testing framework harder to read understand
2) Expose the rig to other weird/unexpected side-effects... maybe for some test in the future, you _want_ that Autosizer behaving nominally vs. being interfered with.

The general idea is that it's a necessary but ugly thing to do, so you should avoid doing it when you can - and injecting it into our Enzyme `main event loop` makes that hard.

I guess at the very least you should add an additional comment inside the .map so the reader can understand when/why that happens.

Reply:
lets save this discussion into a ticket or in-code docs and defer deciding until later.